trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationArrivalDistanceService.java

```diff
@@ -26,10 +26,9 @@ public boolean isApplicable(Station station) {
 
     @Override
     public void request(Station station) {
-        StationDataRequest stationDataRequest = new StationDataRequest();
-        stationDataRequest.setStationName(station.getName());
-        stationDataRequest.setSystemName(station.getSystem().getName());
-
+        StationDataRequest stationDataRequest = new StationDataRequest(
+                null, station.getName(), station.getSystem().getName()
```

Comment:
requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here

Reply:
I was merely keeping things "functionally unchanged" - the old code didn't set requesting module. I'll look into changing it to an enum (ed-pilots-network/backend-util#3) and passing it to the constructors.